frontend/src/pages/Login.tsx

```diff
@@ -66,12 +66,12 @@ export default class LoginForm extends React.Component<LoginProps, LoginState> {
 
   render() {
     return (
-      <Container fluid={true}>
+      <Container className="homeBody" fluid={true}>
         <Row>
-          <Col>
+          <Col className="homeBody">
```

Comment:
@vong98 we should avoid using homebody twice like this, either the first should cascade or we should make homebody CSS class more generic to be used somewhere else (look at the CSS class centre for an example of a generic class). 

I think that homebody should be changed regardless of a more generic class like "content-body"